agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -4554,6 +4544,33 @@ bool backhaul_manager::handle_backhaul_steering_request(ieee1905_1::CmduMessageR
         return false;
     }
 
+    auto is_valid_channel = son::wireless_utils::is_channel_in_operating_class(oper_class, channel);
+
+    if (!is_valid_channel) {
+        LOG(WARNING) << "Invalid channel number";
+
+        LOG(DEBUG) << "Sending ACK message to the originator, mid=" << std::hex << mid;
+        send_cmdu_to_bus(cmdu_tx, controller_bridge_mac, bridge_info.mac);
```

Comment:
This is independent on whether the channel is valid or not, so it can be done before is_valid_channel.

Reply:
Accepted